docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+            "options": {
+                ...
+            },
+        },
+        ...
+    }
+}
+```
+
+A UI object type may or may not have type-specific `options` that configure the rendering of the front-end element.
+
+## 5.1 UI Types
+
+For fields:
+
+- [checkbox](#511-checkbox-element)
+- [date](#512-date-element)
+- [number](#513-numeric-element)
+- [radio](#514-radio-button-element)
+- [select](#515-select-element)
+- [text](#516-text-element)
+- [textarea](#517-textarea-element)
+- [time](#518-time-element)
+
+For helpers:
+
+- [header](#519-header-element) (from h1 to h6)
+- [link](#5110-link-element)
+- [static](#5111-static-element)
+
+These are all mappable to some LORIS React Form element, except for "header" which is native to HTML.
+
+### 5.1.1 Checkbox Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "checkbox"
+        }
+    }
+}
+```
+
+### 5.1.2 Date Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "date"
+        }
+    }
+}
+```
+
+### 5.1.3 Numeric Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "number"
+        }
+    }
+}
+```
+
+### 5.1.4 Radio Button Element
+
+```js
+{
+    "ui": {
+        ...
+        "nameOfRadioButtons" : {
+            "type" : "radio",
+            "options" : {
+                "vertical" : boolean
+            }
+        }
+    }
+}
+```
+
+`options.vertical`: Boolean. True or false that the radio group is rendered vertically. The default value is false, or
+horizontally. 
+
+### 5.1.5 Select Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "select"
+        }
+    }
+}
+```
+
+### 5.1.6 Text Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "text",
+            "options": {
+                "placeholder": string
+            }
+        }
+    }
+}
+```
+
+`options.placeholder`: String. Text that is rendered inside the text element as a placeholder before user input.
+
+### 5.1.7 Textarea Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "textarea",
+            "options": {
+                "placeholder": string
+            }
+        }
+    }
+}
+```
+
+`options.placeholder`: String. Text that is rendered inside the text element as a placeholder before user input.
+
+### 5.1.8 Time Element 
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "time"
+        }
+    }
+}
+```
+
+### 5.1.9 Header Element
+
+A header element renders a "static" `helper` that should be displayed with some level of prominence.
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "header",
+            "options": {
+                "level": int
+            }
+        }
+    }
+}
+```
+
+`options.level`: Int. The number indicating the level of the html heading element to be rendered i.e. h1, the highest
+and most prominent, and h6, the lowest and reserved for subheadings.
```

Comment:
Why link the levels specifically HTML? Are they required to be consecutive? I think it would be better to require the level to be semantic and just say it's recommended that it not go more than 6 levels deep to make it easier to map to html.